scripts/update-apk.sh

```diff
@@ -26,17 +26,21 @@ fi
 find ../app/build/outputs -type f -name '*.apk' -exec cp -v {} . \;
 find ../app/build/outputs -type f -name '*.aab' -exec cp -v {} . \;
 
-if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-    for file in app*; do
-          cp $file eventyay-organizer-master-${file%%}
-    done
-fi
+for file in app*; do
 
-if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
-    for file in app*; do
-          mv $file eventyay-organizer-dev-${file%%}
-    done
-fi
+    if [ ${file} =~ "unaligned" ]; then
```

Comment:
unsigned and unaligned files are not generated now

Reply:
Updated.